### PR TITLE
[WIP] New content - Practice Event Storming and Process Modeling

### PR DESCRIPTION
**Pull request by anthonyvetter**
Replaces old PR 943

### CODE REVIEW

**Comment by anthonyvetter** [comment]
Replacing [PR 943](https://github.com/vmware-tanzu/tanzu-dev-portal/pull/943) with this one. The older PR had too many ports in and was still failing. This is just cleaner :). @joemoore the formatting of this piece looks off. Can you take a look? I'll ping you and @KJefferySmartPV on slack too.

File: content/practices/event-storming-process-modelling/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+   - There are typically 3 roles:
+     - A Narrator reads through the process from left to right.
+     - The Audience listens to the narrator and assists in discovering the process.
+     - A Scribe records the process on the board.
+   - Any participant can take on any of these roles. Rotating roles should also be encouraged, so everyone develops both ownership of the process and a deeper familiarity with the exercise.
+
+   Attempt to narrate the stickies on the board, describing the process of events as they happen from left to right. While you can add grammar in the first narration it is not necessary. The primary goal is to start from en entry point into the system and model the process until a final output that is agreed to be within the scope of your project.
+
+   ##### Grammar:
+
```
Comment (joemoore) [inline content/practices/event-storming-process-modelling/index.md:101]:
```suggestion
      {{< table "table" >}}
```

Comment (joemoore) [reply content/practices/event-storming-process-modelling/index.md:101]: Try wrapping table in the `table` wrapper

File: content/practices/event-storming-process-modelling/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      | <tr><td colspan=3> ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-readmodels.jpg) </td></tr> |
+      | <h4>Systems</h4> | Represent 3rd party tools such as; email, excel, notepad, or any required commercial off the shelf (COTS) applications. | <ul> <li>Systems used by multiple Actions can be duplicated for each Action. You can also add tools developed in-house, that are outside of the scope of the project you are working on</li> </ul>  |
+      | <tr><td colspan=3> ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-systems.jpg) </td></tr> |
+
+      | Legend |  |
+      | --- | --- |
+      | ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-legend.jpg) | ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-overview.jpg) |
+
```
Comment (joemoore) [inline content/practices/event-storming-process-modelling/index.md:120]:
```suggestion
      {{< /table >}}
```

Comment (joemoore) [reply content/practices/event-storming-process-modelling/index.md:120]: Wrapping the table

File: content/practices/event-storming-process-modelling/index.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+   - There are typically 3 roles:
+     - A Narrator reads through the process from left to right.
+     - The Audience listens to the narrator and assists in discovering the process.
+     - A Scribe records the process on the board.
+   - Any participant can take on any of these roles. Rotating roles should also be encouraged, so everyone develops both ownership of the process and a deeper familiarity with the exercise.
+
+   Attempt to narrate the stickies on the board, describing the process of events as they happen from left to right. While you can add grammar in the first narration it is not necessary. The primary goal is to start from en entry point into the system and model the process until a final output that is agreed to be within the scope of your project.
+
+   ##### Grammar:
+      {{< table "table" >}}
```
Comment (joemoore) [inline content/practices/event-storming-process-modelling/index.md:101]:
```suggestion
   {{< table "table" >}}
```

File: content/practices/event-storming-process-modelling/index.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+      | <tr><td colspan=3> ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-readmodels.jpg) </td></tr> |
+      | <h4>Systems</h4> | Represent 3rd party tools such as; email, excel, notepad, or any required commercial off the shelf (COTS) applications. | <ul> <li>Systems used by multiple Actions can be duplicated for each Action. You can also add tools developed in-house, that are outside of the scope of the project you are working on</li> </ul>  |
+      | <tr><td colspan=3> ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-systems.jpg) </td></tr> |
+
+      | Legend |  |
+      | --- | --- |
+      | ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-legend.jpg) | ![This is the placeholder for process modelling grammar](images/process-modelling-grammar-overview.jpg) |
+      {{< /table >}}
```
Comment (joemoore) [inline content/practices/event-storming-process-modelling/index.md:120]:
```suggestion
   {{< /table >}}
```

File: content/practices/event-storming-process-modelling/index.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
```
Comment (joemoore) [inline content/practices/event-storming-process-modelling/index.md]:
```suggestion
   {{< /table >}}
```

**Comment by joemoore** [comment]
**_TL;DR:_** I don't think this should be a Practice; instead I think it is more appropriate as a [Guide](https://tanzu.vmware.com/developer/guides/), [Blog Post(s)](https://tanzu.vmware.com/developer/blog/), or another format that does not constrain the structure of this content unnaturally and force the author(s) to adapt the content to a restrictive structure. Doing so does the author(s) and content a disservice. I suggest revisiting this content within the context of a Guide or Blog Post(s) and see if this enables it to be more useful and consumable without the constraints of the Practice's strict format.

**Details:** 
I have fixed the formatting. That said, I felt unclear how to move forward. Unlike the other practices I feel overwhelmed and confused by this content, and there are still several sections that I find out of place, which I can annotate in the file content-diff itself if needed. But I think I figured out the issue: those issues are really symptoms that we are trying to force this content into the "Practices" format when it should likely be something else, such as a Guide, Blog Post or series of them, or similar. These symptoms include: 
- The "Why do it" section is 2x -3x longer than another other practice.
- The content itself is 2x -3x longer and more detailed than any other practice.
- There is a very large "History, Background, and Context" section just to set the scene for the facilitator before the reader can even get started.
- There is a very large (and needed) Grammar definition section, without which the facilitator and cannot succeed.
- The "Success & Expected Outcomes" section contains even more activities to do, and definitions of things.
- ... after which there yet another section of notes that defines even more definitions and concepts! 
- ... after which there is a "Related Practices" section with 11(!!) related practices and activities.

Much of context-setting and concept-defining content mentioned above feels out of place because it doesn't "have a home" in the typical Practice template, as this amount of contest is not needed for other sessions. Yes, other Practices have context-setting and definitions, but this feels more like trying to distill an entire book and years worth of experience into a single session, and it's just to big and too much. This is not the fault of the author(s) or the content, but a manifestation of the content being forced into the wrong structure. 

My suggestion is to reimagine the content as a Guide or series of blog posts to it can take a more flexible format and have as much context and history-setting it needs to be as impactful as possible. This might the be the case with the other Event Storming and SWIFT-themed practices, too. I've received feedback that the [Boris practice](https://tanzu.vmware.com/developer/practices/boris/) is nothing like the actual experience. I think it is unrealistic for any of these sessions to be a Practice that a facilitator can pull up before a session and use as a guide for the next hour or so. They're just too big, too involved, too complicated -- they're different beasts, and that's okay. 

Thoughts?
